.github/workflows/on_pull_request.yaml

```diff
@@ -12,6 +12,6 @@ jobs:
       pull-requests: read
     secrets: inherit
     with:
-      microk8s-channel: 1.26/stable
+      microk8s-channel: 1.26-strict/stable
```

Comment:
I see you adding this in a few places.  is there a reason why we shouldn't use `1.26-strict/stable` as the workflow default arg?  I think we'll almost always use 1.26-strict, juju 3.1/stable, and python 3.8 - why not make those the workflow's defaults and remove all these inputs?

Reply:
Sorry, forgot to revert those changes after sending https://github.com/canonical/pipelines-rocks/pull/58.